.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ on: [push, pull_request]
 jobs:
   test:
     name: Test
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
Should we just do `ubuntu-latest`?

Reply:
Probably not, because it's just as likely to break OTP version compatibility I think.

Reply:
@ericmj we'll likely keep updating OTP though, so I think it would be one less thing to keep track of.